fulp_modules/features/species/beefmen/beefman_clothing.dm

```diff
@@ -13,7 +13,7 @@
 //Captain
 /obj/item/clothing/under/bodysash/captain
 	name = "captain's sash"
-	desc = "A sash with gold markings denoting the rank of \"Captain\", slung from shoulder to hip."
+	desc = "A gold-embroidered sash with markings that denote its prestigious wearer as Captain."
```

Comment:
Keep the quotation marks

Reply:
_See response to the second suggestion below this one._

Reply:
The job title is a label so it can go with or without quotation marks, now it just seems sarcastic/demeaning

Reply:
I don't understand what you mean in regards to labels being valid for optional quotation. 

Also the original text seems more demeaning since it calls the captain a "captain" in the same way you might call a doctor of nihilistic medicine a "doctor" or a lawyer of Ancient Sumerian law a "lawyer".